### PR TITLE
[WIP] add bank-account

### PR DESCRIPTION
**Pull request by angyangie**
We will be working on bank-account now.

### CODE REVIEW

**Comment by bernardoamc** [comment]
Hi @angyangie! This project uses [rubocop](https://github.com/bbatsov/rubocop). Rubocop will analyze your code and see if it is following a specific guideline. You can see the complains by clicking on the `Details` button.

screen shot 2016-05-13 at 3 50 43 pm

**Comment by kotp** [comment]
If you would like to use the bin/local-status-check or even set up the hook for this, there is a tool for that in bin as well.

**Comment by angyangie** [comment]
Thank you for the help. I'm looking at robocop now, and fixing errors.
Weirdly, I am having to fix robocop issues with files I didn't edit (other
problems). Do you know why this may be? These are all spacing/syntax
issues.

**Comment by bernardoamc** [comment]
>  I am having to fix robocop issues with files I didn't edit (other problems). Do you know why this may be? 

There are a few files that are not related with this solution in your pull request, you can try to remove them. :) For example: `exercises/bowling/bowling_test.rb`

**Comment by kotp** [comment]
Make sure you rebase your solution based on the latest master branch of exercism/xruby.  You may be able to do `git rebase xruby/master` if 'xruby' points to the exercism/xruby name.

File: exercises/bowling/bowling_test.rb [hunk 1]
```diff
@@ -1,226 +0,0 @@
...
-    end
-  end
-  private :roll_n_times
-
-  # Don't forget to define a constant VERSION inside of Game.
-  def test_bookkeeping
-    skip
-    assert_equal 1, Game::VERSION
-  end
-end
```
Comment (tute) [inline exercises/bowling/bowling_test.rb]: This seems to be a change that doesn't belong in this PR.

File: exercises/bowling/example.rb [hunk 2]
```diff
@@ -1,89 +0,0 @@
-class Game
```
Comment (tute) [inline exercises/bowling/example.rb]: > This seems to be a change that doesn't belong in this PR.

This too

**Comment by kytrinyx** [comment]
@angyangie You might also have a different version of rubocop -- we've had some trouble with that in the past.

**Comment by kotp** [comment]
`bundle install` and `bundle exec rubocop` should address those issues.

**Comment by Insti** [comment]
Thanks for the effort you've put into this.
Since you seem to have stopped working on it before it was completed,
we're closing this request without merging in the changes.

If you decide you want to continue and complete it, please open a new pull
request and we'll be happy to assist you.

**Comment by kytrinyx** [comment]
> If you decide you want to continue and complete it, please open a new pull
> request and we'll be happy to assist you.

@Insti How about suggesting they can re-open it? That might be more complex, though, because we'd have to help them rebase.
